plan/plan.go

```diff
@@ -164,8 +166,14 @@ type PhysicalPlan interface {
 	json.Marshaler
 	Plan
 
-	// matchProperty means that this physical plan will try to return the best plan that matches the required property.
-	// childPlanInfo means the plan infos returned by children.
+	// matchProperty calculates the cost of the physical plan if it matches the required property.
+	// It's usually called at the end of convert2PhysicalPlan, some PhysicalPlan does not implement it as there is
+	// no property to match, they just do the cost calculation directly.
```

Comment:
It's usually called at the end of convert2PhysicalPlan. If some physical plans do not implement it because there is no property to match, these plans just do the cost calculation directly.